gnovm/pkg/gnolang/helpers_test.go

```diff
@@ -53,3 +53,103 @@ func TestIsStdlib(t *testing.T) {
 		)
 	}
 }
+
+func TestIsEphemeralPath(t *testing.T) {
+	tests := []struct {
+		name     string
+		pkgPath  string
+		expected bool
+	}{
+		{
+			name:     "valid ephemeral path",
+			pkgPath:  "gno.land/e/user123/test",
+			expected: true,
+		},
+		{
+			name:     "valid ephemeral run path",
+			pkgPath:  "gno.land/e/g1user123/run",
+			expected: true,
+		},
+		{
+			name:     "valid ephemeral path with subdirectories",
+			pkgPath:  "gno.land/e/user123/subdir/test",
+			expected: true,
```

Comment:
It could be relevant to mention these two different forms of ephemeral paths in the documentation.